lib/compat/wordpress-6.4/fonts/font-face/class-wp-font-face-resolver.php

```diff
@@ -56,8 +56,8 @@ private static function parse_settings( array $settings ) {
 		foreach ( $settings['typography']['fontFamilies'] as $font_families ) {
 			foreach ( $font_families as $definition ) {
 
-				// Skip if font-family "name" is not defined.
-				if ( empty( $definition['name'] ) ) {
+				// Skip if font-family "slug" is not defined.
+				if ( empty( $definition['slug'] ) ) {
```

Comment:
I don't think it should use `slug`. Instead I think it should use `fontFamily`. Why? `slug` is not the proper name of the Font Family, but rather is the lowercase and hyphen version of the name. Currently Font Face uses this font-family name in the CSS.

I have PR coming shortly to show it in action.

Reply:
PR https://github.com/WordPress/gutenberg/pull/54615 uses the `fontFamily` setting as the fallback to get the name.